src/Node/BaseNode.php

```diff
@@ -225,6 +225,10 @@ public function getValue(string $field, $value)
 
     public function walk($input)
     {
+        if (!is_array($input)) {
+            return $input;
+        }
+
         $result = [];
 
         if (!$this->hasChildren()) {
```

Comment:
I was talking about this validation too, this was before, but since you are here it may help the code

Reply:
Done mate! Thanks!